web/src/games/cardtable/game.test.ts

```diff
@@ -3,113 +3,113 @@ import { Local } from 'boardgame.io/multiplayer';
-    expect(G.deck[1]).toEqual({ id: cardEnum.C2, rank: 15, faced: false });
-    expect(G.deck[2]).toEqual({ id: cardEnum.DJ, rank: 37, faced: false });
-    expect(G.deck[3]).toEqual({ id: cardEnum.C10, rank: 23, faced: false });
-  });
-
-  test('cutDeck(5) should rotate deck by one card', () => {
-    let client = Client({ game: { ...CardTableGame, seed: 327 }, playerID: '0', multiplayer: Local() });
-    client.start();
-    client.events.setPhase(null);
-    client.moves.cutDeck(5);
-    const { G } = client.store.getState();
-    expect(G.deck[0]).toEqual({ id: cardEnum.C7, rank: 20, faced: false });
-    expect(G.deck[1]).toEqual({ id: cardEnum.C6, rank: 19, faced: false });
-    expect(G.deck[2]).toEqual({ id: cardEnum.D10, rank: 36, faced: false });
-    expect(G.deck[3]).toEqual({ id: cardEnum.D5, rank: 31, faced: false });
-  });
-});
-
-describe('moveCard(ICardMove) consistent state changes to G', () => {
-  test('move from deck[0] to deck[1]', () => {
-    let client = Client({ game: { ...CardTableGame, seed: 327 }, playerID: '0', multiplayer: Local() });
-    client.start();
-    client.events.setPhase(null);
-    client.moves.moveCards({
-      from: { container: 'deck', ordinal: 0 },
-      to: { container: 'deck', ordinal: 1 },
-    });
-    const { G } = client.store.getState();
-    expect(G.deck[0].id).toEqual(cardEnum.SK);
-    expect(G.deck[1].id).toEqual(cardEnum.D6);
-  });
-
-  test('move from deck[0-2] to hands.east.private[0]', () => {
-    let client = Client({ game: { ...CardTableGame, seed: 327 }, playerID: '0', multiplayer: Local() });
-    client.start();
-    client.events.setPhase(null);
-    client.moves.moveCards({
-      from: { container: 'deck', ordinal: 0, cardcount: 3 },
-      to: { container: 'hands.east.private', ordinal: 0 },
-    });
-    const { G } = client.store.getState();
-    expect(G.deck.length).toEqual(49);
-
-    expect(G.hands.east.private).toEqual([
-      {
-        id: cardEnum.D6,
-        rank: 32,
-        faced: false,
-      },
-      {
-        id: cardEnum.SK,
-        rank: 13,
-        faced: false,
-      },
-      {
-        id: cardEnum.SJ,
-        rank: 11,
-        faced: false,
-      },
-    ]);
-  });
-});
+// describe('deck moves', () => {
+//   test('cutDeck(1) should rotate deck by one card', () => {
+//     let client = Client({ game: { ...CardTableGame, seed: 327 }, playerID: '0', multiplayer: Local() });
+//     client.start();
+//     client.events.setPhase(null);
+//     client.moves.cutDeck(1);
+//     const { G } = client.store.getState();
+
+//     expect(G.deck[0]).toEqual({ id: cardEnum.SK, rank: 13, faced: false });
+//     expect(G.deck[1]).toEqual({ id: cardEnum.SJ, rank: 11, faced: false });
+//     expect(G.deck[2]).toEqual({ id: cardEnum.H9, rank: 48, faced: false });
+//     expect(G.deck[3]).toEqual({ id: cardEnum.H2, rank: 41, faced: false });
+//   });
+
+//   test('cutDeck(-3) should rotate deck by one card', () => {
```

Comment:
Why are you commenting out all these tests? Are they broken? Can you fix them?

Reply:
They are emitting errors that I've mentioned on bgio and fbg discord.  The errors appear benign but I was trying to isolate and investigate their cause.  I'll add the tests back -- this was a miss.

Reply:
added back the tests